Samples/AcnDataMatrix/MainWindow.xaml.cs

```diff
@@ -110,9 +110,50 @@ private void btnStartApp_Click(object sender, RoutedEventArgs e)
             aTimer.Start();            
         }
 
+        /// <summary>
+        /// A list of the Sync IP address that we have seen. We wait for a sync packet from all of these before rendering
+        /// </summary>
+        List<string> SyncIps = new List<string>();
```

Comment:
Given how these are used, wouldn't these be much more efficient as `HashSet<IpEndPoint>`?